Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -1385,25 +1387,28 @@ std::pair<u32, u32> GetDispatchCount(const DecodingShaderInfo* info, u32 width,
           (height + (info->group_size_y - 1)) / info->group_size_y};
 }
 
-std::string GenerateDecodingShader(TextureFormat format, TLUTFormat palette_format,
+std::string GenerateDecodingShader(TextureFormat format, std::optional<TLUTFormat> palette_format,
                                    APIType api_type)
 {
   const DecodingShaderInfo* info = GetDecodingShaderInfo(format);
   if (!info)
     return "";
 
   std::ostringstream ss;
-  switch (palette_format)
+  if (palette_format.has_value())
```

Comment:
I don't have a strong preference either way but in case you didn't know, you can do:

`if (palette_format)`

Reply:
I think I'll stick with `.has_value()` here, since we're not really space constrained